plugin/plugins/mijia/__init__.py

```
@@ -0,0 +1,1205 @@
+        old_api = self.api
+        self.api = new_api
+        if old_api is not None:
+            try:
+                await old_api.close()
+            except Exception as close_err:
+                self.logger.warning(f"关闭旧API客户端时出错: {close_err}")
+        
+        self.logger.info("API客户端初始化成功")
+
+    async def _reload_credential(self):
+        """重新加载凭据（如配置变化）"""
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)